RxCocoa/iOS/UICollectionView+Rx.swift

```diff
@@ -179,6 +179,8 @@ extension UICollectionView {
 }
 
 extension Reactive where Base: UICollectionView {
+    public typealias DisplayCollectionViewCellEvent = (cell: UICollectionViewCell, indexPath: IndexPath)
+    public typealias DisplayCollectionReusableViewEvent = (reuseableView: UICollectionReusableView, elementKind: String, indexPath: IndexPath)
```

Comment:
I think that for consistency sake this should be named `DisplayCollectionViewSupplementaryViewEvent = (supplementaryView: UICollectionReusableView, ...` and `at: IndexPath)` because this is it's name in all delegate methods.
`UICollectionReusableView` is it's type.